### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by hijkzzz** [OpenRLHF/OpenRLHF#568]

**Comment by zhaochenyang20** [comment]
Could you @ me for SGLang support? Thanks! @hijkzzz

**Comment by UbeCc** [comment]
I'm working on sft + ring-attn thanks! @hijkzzz

**Comment by catqaq** [comment]
More discussion: 
I personally pay great attention to the numbers 9 and 4. Friends who are interested can get in touch with me.

<img width="641" alt="image" src="https://github.com/user-attachments/assets/eab69d96-b971-41ef-8935-d5ce5b487ffc">

**Comment by zhaochenyang20** [comment]
> More discussion: I personally pay great attention to the numbers 9 (moe related) and 4. Friends who are interested can get in touch with me.
> 
> <img alt="image" width="641" src="https://private-user-images.githubusercontent.com/42762740/394348123-eab69d96-b971-41ef-8935-d5ce5b487ffc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4ODgyODIsIm5iZiI6MTczMzg4Nzk4MiwicGF0aCI6Ii80Mjc2Mjc0MC8zOTQzNDgxMjMtZWFiNjlkOTYtYjk3MS00MWVmLTg5MzUtZDVjZTViNDg3ZmZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjExVDAzMzMwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRhNDAxNDUzNDA1MjM0M2E3NmZkNTExODY3YzQ1MDU2MDkxNmYxMzYwZGNkODE3YzQ2Zjg5NzQxODE5NmQ2ODUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Yp7_5BqIREOZOQ3MwHFiuHmm5YH5mhLFskinRgcUDAc">

Yeah. We now support Deepseek v2.5 1210 model's 230B MOE with FP8 on 8 H100. You can first quantize the model and load it with SGLang to see the speed up.

**Comment by zhaochenyang20** [comment]
https://lmsys.org/blog/2024-12-04-sglang-v0-4/#data-parallelism-attention-for-deepseek-models

@catqaq See this part.

**Comment by hsliuustc0106** [comment]
is there any plan for posttraining on omnimodality models since vLLM-Omni was released recently?
